File: VB/WindowsFormsApplication741/MyFlyoutPanel.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.Utils

Namespace WindowsFormsApplication741
    Public Class MyFlyoutPanel
        Inherits FlyoutPanel

        Protected Overrides Function CreateToolFormCore(ByVal owner As Control, ByVal content As FlyoutPanel, ByVal options As FlyoutPanelOptions) As FlyoutPanelToolForm
            Return New MyFlyoutPanelToolForm(owner, content, options)
        End Function
    End Class
End Namespace

File: CS/WindowsFormsApplication741/MyFlyoutPanel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.Utils;

namespace WindowsFormsApplication741 {
    public class MyFlyoutPanel : FlyoutPanel {
        protected override FlyoutPanelToolForm CreateToolFormCore(Control owner, FlyoutPanel content, FlyoutPanelOptions options) {
            return new MyFlyoutPanelToolForm(owner, content, options);
        }
    }
}
